tslint.ts

```diff
@@ -155,6 +155,9 @@ const rules : tslint.Configuration.RawRulesConfig = Object.assign({}, rulesAirbn
     groups: importGroups,
   }],
 
+  // for the tslint-react-hooks plugin
+  'react-hooks-nesting': "error"
```

Comment:
could it be `true` instead of `"error"` - so that it takes the globally configured severity? Does the plugin require this to be a non-standard value like that?

Reply:
I changed it to true, however according to the official [docs of tslint](https://palantir.github.io/tslint/usage/configuration/) severity looks like this:

> severity?: "default" | "error" | "warning" | "off"
> Legacy: A boolean value may be specified instead of the above object, and is equivalent to setting no options with default severity.

We can update that at another time if we want.

Reply:
okay if `true` works for this rule, then it's fine